src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/ConnectionTest.php

```diff
@@ -266,7 +267,7 @@ public function testItSetupsTheConnectionWithDefaults()
         );
 
         $amqpExchange->expects($this->once())->method('declareExchange');
-        $amqpExchange->expects($this->once())->method('publish')->with('body', null, AMQP_NOPARAM, ['headers' => [], 'delivery_mode' => 2]);
+        $amqpExchange->expects($this->once())->method('publish')->with('body', null, AMQP_NOPARAM, ['headers' => [], 'delivery_mode' => 2, 'timestamp' => ClockMock::time()]);
```

Comment:
Instead of using ClockMock directly, you should use a `@group time-sensitive` annotation on the test class itself.

Reply:
Yeah I see now, fixed :)